tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py27
+
+[testenv]
+deps = 
+    -rrequirements.txt
+    pytest
+    ansible
```

Comment:
We should create a `requirements-dev.txt` file and have pytest, ansible, tox in that as well as `-r requirements.txt.